newrelic.yml

```diff
@@ -403,6 +403,10 @@ common: &default_settings
   # prepend, chain, disabled.
   # instrumentation.bunny: auto
 
+# Controls auto-instrumentation of DynamoDb at start-up.
+# May be one of [auto|prepend|chain|disabled]
+# instrumentation.dynamodb: auto
```

Comment:
The indentation on these new lines is off. You can either indent a bit further to fix it or revert the changes to this file and let the auto-generated changes to the file handle it later when we release.

Reply:
ahh yeah that's the auto generate command doing that, since this is automated later on now we can probably just remove this from the scaffold command. 
this indentation fixed now though